docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -355,3 +391,65 @@ When using the code editor, after clicking on *Submit*, the text *Building and t
 Bamboo → Artemis
 ^^^^^^^^^^^^^^^^^
 The build result is displayed in the code editor.
+
+Deployment on Kubernetes
+------------------------
+
+This part of the documentation assumes you have a Kubernetes cluster running, as well as DockerHub account, if not you can find how to set it up here TO DO `<>`__
```

Comment:
Looks like a small unfinished TO DO here. I am assuming you just missed it out.

Reply:
I haven't added the link since it will follow a new page which will be invalid at the moment but I will write a comment explaining that